resources/windows/wix/directorylist.wxs.erb

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <Fragment>
+
+  <%- @project.generate_wix_dirs(self) -%>
```

Comment:
p.s., this needs to be nested under `<Directory Id='TARGETDIR' Name='SourceDir' >`

Also, this still needs to start with `<%=` otherwise we aren't going to get any output